src/acl.js

```diff
@@ -0,0 +1,22 @@
+const telegramUtility = require("./util/telegram");
+
+const groupAdminMiddleware = ({ bot }) => async (ctx, next) => {
+  const chatId = telegramUtility.chatIdInContext(ctx);
+  const fromId = telegramUtility.fromIdInContext(ctx);
+
+  if (telegramUtility.chatInContextIsGroupChat(ctx)) {
+    const chatAdmins = await bot.telegram.getChatAdministrators(chatId);
+    ctx.fromIsGroupAdmin = chatAdmins.some((admin) => admin.user.id == fromId);
+  } else {
+    ctx.fromIsGroupAdmin = true;
+  }
+  next(ctx);
+};
+
+const groupAdminCommandTransformer = (cmd) => (ctx, next) => {
```

Comment:
Let's give this a more concise name. `acl.groupAdminCommandTransformer(commands.start())` in the other file is pretty long and doesn't read that well.

what do you thing of `acl.adminOnly`? Usage would then look like this:

```javascript
bot.start(acl.adminOnly(commands.start()));
```